resources/languages/nl/corpus/time.clj

```diff
@@ -0,0 +1,322 @@
+(
+  ;Context map
+  ; Tuesday Feb 12, 2013 at 4:30am is the "now" for the tests
+  {:reference-time (time/t -2 2017 1 27 4 30)}
+
+  ; dagen
+  "nu"
+  "op dit moment"
+  "zo snel mogelijk"
+  (datetime 2017 1 27 4 30 00)
+
+  "vandaag"
+  "momenteel"
+  (datetime 2017 1 27)
+
+  "gisteren"
+  (datetime 2017 1 26)
+
+  "morgen"
+  (datetime 2017 1 28)
+
+  "morgen avond"
```

Comment:
I would like to see here "morgenavond" as well, that's the more common type of writing.